pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[tool.poetry]
+name = "climate-assessment"
+version = "0.1.4a0"
```

Comment:
I always like using the poetry dynamic versioning plugin that auto generates a version based on the commit.

Reply:
I'm on the fence. It feels like an unnecessary extra dependency in the tool chain to me for the most part so I tried it then stopped. Happy if you want to introduce it in a follow up PR though.

Reply:
Fair enough, it does add an extra dependency.